packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx

```diff
@@ -13,19 +13,23 @@
 import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
 import { memo, ReactElement, useMemo } from "react";
 
-import { createTheme, deepmerge, ThemeOptions } from ".";
-import { odysseyTheme } from "./theme";
+import { createTheme, deepmerge, ThemeOptions, TokenOverrideOptions } from ".";
+import { createOdysseyTheme } from "./theme";
 
 const OdysseyThemeProvider = ({
   children,
   themeOverride,
+  tokenOverride,
 }: {
   children: ReactElement;
   themeOverride?: ThemeOptions;
+  tokenOverride?: TokenOverrideOptions;
 }) => {
+  const odysseyTheme = createOdysseyTheme(tokenOverride);
+
   const customOdysseyTheme = useMemo(
     () => themeOverride && createTheme(deepmerge(odysseyTheme, themeOverride)),
-    [themeOverride]
+    [odysseyTheme, themeOverride]
```

Comment:
I see. So there's the Odyssey theme, and then the MUI version of the Odyssey theme. So this `createOdysseyTheme` function should be `createOdysseyMuiTheme`.

`createOdysseyTheme` should return an `OdysseyTheme` object which is essentially just the tokens right?

Reply:
sure, I guess there's no use case for createOdysseyTheme for now where we need to return the tokens, right? I'll only export createOdysseyMuiTheme for now